Cargo.toml

```diff
@@ -217,7 +217,7 @@ alloy-trie = "0.4.1"
 
 ## zksync
 era_test_node = { git="https://github.com/matter-labs/era-test-node.git" , rev = "2041018903aa7e00e74c450f8c0baf799c056d86" }
-zksync-web3-rs = {git = "https://github.com/lambdaclass/zksync-web3-rs.git", rev = "2da644f5b8fc48a129e80fe0653e5334701c059b"}
+zksync-web3-rs = {git = "https://github.com/jrigada/zksync-web3-rs.git", rev = "bc3e6d3e75b7ff3747ff2dccefa9fb74d770931b"}
```

Comment:
Is there a bug that should be reported and fixed upstream?

Reply:
If upstream is unmaintained, we should look into alternatives, and at the least fork it into MSL and use that. 
And preferably ensure the encoding matches https://github.com/matter-labs/zksync-era/blob/main/core%2Flib%2Ftypes%2Fsrc%2Ftransaction_request.rs#L478